discordbot/stocks/technical_analysis/ta_menu.py

```diff
@@ -0,0 +1,916 @@
+                        elif reaction.emoji == "6️⃣":
+                            if cfg.DEBUG:
+                                print("Reaction selected: 6")
+                            if current == 0:
+                                await hma_command(ctx, ticker)
+                            elif current == 1:
+                                await bbands_command(ctx, ticker)
+                        elif reaction.emoji == "7️⃣":
+                            if cfg.DEBUG:
+                                print("Reaction selected: 7")
+                            if current == 0:
+                                await zlma_command(ctx, ticker)
+                            elif current == 1:
+                                await donchian_command(ctx, ticker)
+                        elif reaction.emoji == "8️⃣":
+                            if cfg.DEBUG:
+                                print("Reaction selected: 8")
+                            if current == 0:
+                                await cci_command(ctx, ticker)
+                            elif current == 1:
+                                await kc_command(ctx, ticker)
+                        elif reaction.emoji == "9️⃣":
+                            if cfg.DEBUG:
+                                print("Reaction selected: 9")
+                            if current == 0:
+                                await macd_command(ctx, ticker)
+                            elif current == 1:
+                                await ad_command(ctx, ticker)
+                        for emoji in emoji_list:
+                            await main_message.remove_reaction(emoji, ctx.bot.user)
+                        components = [
+                            [
+                                discord_components.Button(
+                                    label="Prev",
+                                    id="back",
+                                    style=discord_components.ButtonStyle.green,
+                                    disabled=True,
+                                ),
+                                discord_components.Button(
+                                    label=f"Page {int(cols.index(cols[current]))}/{len(cols) - 1}",
+                                    id="cur",
+                                    style=discord_components.ButtonStyle.grey,
+                                    disabled=True,
+                                ),
+                                discord_components.Button(
+                                    label="Next",
+                                    id="front",
+                                    style=discord_components.ButtonStyle.green,
+                                    disabled=True,
+                                ),
+                            ]
+                        ]
+                        await main_message.edit(components=components)
+                        return
+
+            except asyncio.TimeoutError:
```

Comment:
In case of a TimeoutError we prob should consider sending an error message? What do you think?

Reply:
The timeout error happens after the selected timer runs out and makes that the user no longer can press the buttons of the message. So the "error" isn't an real error or problem.

Reply:
True, my point is to send a message saying that. Rather than just not allowing user to select options say: "Error timeout - you snooze you lose! 😋 "

Reply:
Should I append it to the menu message or send a new message?

Reply:
Send a new message I'd say